package day41_errors_garbageCollector;

public class C01_Exceptions {

	public static void main(String[] args) {
	/*	
	Object obj = new Integer(3);
	String str = (String) obj ;
	System.out.println(str);
	
	*/
	/*
	System.out.println(5/0);
	*/
		
		
	}	
		
	
		
		
		
	
	/*
	
		System.out.println(exceptions());
	}

	@SuppressWarnings("finally")
	public static String exceptions() {
		String result ="";
		String v = null ;
		
		try {
			try {
			result = result + "a" ;
			v.length() ;
			result = result + "b";
			
			} catch (NullPointerException e) {
				result = result + "c" ;
				
			}finally {
				result = result + "d" ;
				throw new Exception() ;
			}
		} catch (Exception e) {
			result = result + "e" ;
		}
		
	
		return result;*/
	}


